selfdrive/car/toyota/carcontroller.py

```diff
@@ -49,15 +50,21 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, hud_alert,
     # *** compute control surfaces ***
 
     # gas and brake
-
-    apply_gas = clip(actuators.gas, 0., 1.)
+    apply_gas = 0.
+    apply_accel = actuators.gas - actuators.brake
```

Comment:
think this is more clear:
```suggestion
    interceptor_gas_cmd = 0.
    pcm_accel_cmd = actuators.gas - actuators.brake
```